### PR TITLE
Custom API backend check

### PR DESCRIPTION
**Pull request by assemmarwan**

### CODE REVIEW

**Comment by nabilshuja** [comment]
@assemmarwan I think it's fine to call `loadAppVersions` in the constructor rather.

File: lib/src/core/renovation.dart [hunk 1]
```diff
@@ -124,18 +128,33 @@ class Renovation {
         'Accept': 'application/json',
         'X-Requested-With': 'XMLHttpRequest'
       };
+
+      frappe = Frappe(config);
+      unawaited(frappe.loadAppVersions());
```
Comment (nabilshuja) [inline lib/src/core/renovation.dart]: @assemmarwan I think it's fine to call loadAppVersions in the constructor rather.

File: lib/src/core/renovation.dart [hunk 2]
```diff
@@ -2,6 +2,8 @@ import 'dart:core';
 
 import 'package:logger/logger.dart';
 import 'package:pedantic/pedantic.dart';
+import 'package:renovation_core/core.dart';
+import 'package:renovation_core/src/core/errors.dart';
```
Comment (nabilshuja) [inline lib/src/core/renovation.dart]: @assemmarwan maybe export the `errors.dart` within `core.dart`, what do you think?

**Review by nabilshuja — CHANGES_REQUESTED** [review]

File: lib/src/core/renovation.dart (hunk 2 above)
Comment (assemmarwan) [reply lib/src/core/renovation.dart]: Actually I should import using relative reference. Good catch 👍

**Review by nabilshuja — APPROVED** [review]

**Review by nabilshuja — APPROVED** [review]

**Pull request merged**